src/api/config/cache.go

```diff
@@ -0,0 +1,56 @@
+package config
+
+import (
+	"sync"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/albrow/zoom"
+)
+
+const defaultHost = "localhost:6379"
+
+// Pool is a pool of Zoom connections used by other packages
+var pool *zoom.Pool
+
+var once sync.Once
+
+type redisConfig struct {
+	Host string
+}
+
+// GetRedisPool returns a pool of Zoom connections
+func GetRedisPool() *zoom.Pool {
+	once.Do(func() {
+		pool = newRedisPool()
+	})
+	return pool
+}
+
+// CloseRedisPool closes a pool of Zoom connections
+func CloseRedisPool() {
```

Comment:
I was considering adding a mutex here, but I agree that the easier option is to just not close the pool.